src/olympia/blocklist/admin.py

```diff
@@ -141,14 +129,75 @@ class MultiBlockSubmitAdmin(admin.ModelAdmin):
     )
     ordering = ['-created']
     view_on_site = False
-    list_select_related = ('updated_by',)
+    list_select_related = ('updated_by', 'signoff_by')
+    change_form_template = 'blocklist/block_submission_change_form.html'
+
+    class Media:
+        css = {
+            'all': ('css/admin/blocklist_blocksubmission.css',)
+        }
 
     def has_delete_permission(self, request, obj=None):
+        # For now, keep all BlockSubmission records.
+        # TODO: define under what cirumstances records can be safely deleted.
```

Comment:
Is there an issue filed for this `TODO`? If not, I suggest creating one.

Reply:
https://github.com/mozilla/addons-server/issues/13278